<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>profil_gerant</class>
 <widget class="QDialog" name="profil_gerant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1272</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ButtonProfil">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;border:2px solid black;
background-color: rgb(204, 204, 204);</string>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gerer_rupture_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>540</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Statistiques</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gerer_utilisateurs">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Gérer Les utilisateurs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_accueilgerat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Accueil</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>16</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/R (1).png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>320</y>
     <width>37</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>951</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);border:2px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gere_vente">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>240</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(204, 204, 204);;font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Gérer les ventes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/WhatsApp Image 2024-04-30 à 20.55.44_6b270705.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nome_user">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
font: 900 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gerer_fournisseur">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Gérer Les fournisseurs</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>20</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);font-size:20px;
font: 20pt &quot;Segoe Script&quot;;border:2px solid black;
color: rgb(12, 255, 125);</string>
   </property>
   <property name="text">
    <string>PHARMACIE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>61</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/devenir-pharmacien-921x1024.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_quitter">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>10</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid black;color:white;border-radius:10px;
font: 900 12pt &quot;Arial Black&quot;;
background-color: rgb(255, 114, 116);</string>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/OIP (3).jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid blue;border-radius:4px;
font: 14pt &quot;Rockwell&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);font-size:15px;
font: 600 16pt &quot;Segoe UI Variable Text Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>Dr.</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>-10</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gerer_stock">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 204, 204);font-size:15px;border:2px solid black;</string>
   </property>
   <property name="text">
    <string>Gérer le stock</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>16</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_aller_a_modifier_gerant">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>540</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 800 20pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(96, 156, 18);;color:white;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>721</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(227, 227, 227);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_nom_gerant">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:10px;font: 18pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_prenom_gerant">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:10px;font: 18pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
background-color: rgb(7, 255, 106);
color: rgb(255, 254, 211);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>   Prenom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numero_gerant">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:10px;font: 18pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_etat_gerant">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:10px;font: 18pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_identifiant_gerant">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:10px;font: 18pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
background-color: rgb(7, 255, 106);
color: rgb(255, 254, 211);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>  Numero</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
background-color: rgb(7, 255, 106);
color: rgb(255, 254, 211);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>    Etat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
background-color: rgb(7, 255, 106);
color: rgb(255, 254, 211);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>  Identifiant</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
background-color: rgb(7, 255, 106);
color: rgb(255, 254, 211);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>    Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_email_gerant">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);border-radius:10px;font: 12pt &quot;Segoe Fluent Icons&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;Baskerville Old Face&quot;;
background-color: rgb(7, 255, 106);
color: rgb(255, 254, 211);border-radius:10px;</string>
    </property>
    <property name="text">
     <string>    Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Rockwell&quot;;
background-color: rgb(255, 255, 255);
border:2px solid rgb(235, 15, 255);
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>     Votre Profile</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Annule_gerant">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>540</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(53, 125, 218);color:white;border-radius:10px;
font: 900 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>160</y>
     <width>141</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressource.qrc">:/res/image/R.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>ButtonProfil</zorder>
  <zorder>pushButton_gerer_rupture_2</zorder>
  <zorder>pushButton_gerer_utilisateurs</zorder>
  <zorder>pushButton_accueilgerat</zorder>
  <zorder>line_2</zorder>
  <zorder>label_7</zorder>
  <zorder>label</zorder>
  <zorder>pushButton_gere_vente</zorder>
  <zorder>label_4</zorder>
  <zorder>nome_user</zorder>
  <zorder>pushButton_gerer_fournisseur</zorder>
  <zorder>line_4</zorder>
  <zorder>label_2</zorder>
  <zorder>label_8</zorder>
  <zorder>pushButton_quitter</zorder>
  <zorder>label_5</zorder>
  <zorder>label_date</zorder>
  <zorder>label_6</zorder>
  <zorder>line</zorder>
  <zorder>pushButton_gerer_stock</zorder>
  <zorder>line_3</zorder>
  <zorder>pushButton_aller_a_modifier_gerant</zorder>
  <zorder>groupBox</zorder>
  <zorder>pushButton_Annule_gerant</zorder>
  <zorder>label_14</zorder>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
